core/src/com/goonsquad/galactictd/systems/game/WarpGeneratingSystem.java

```diff
@@ -31,6 +31,6 @@ private void generateWarpLocation() {
         warpLocation.x = MathUtils.random(warpWidth, PlayScreen.GAME_WIDTH - warpWidth);
         warpLocation.y = MathUtils.random(
                 warpHeight,
-                PlayScreen.GAME_HEIGHT - PlayScreen.HUD_HEIGHT - warpHeight);
+                PlayScreen.GAME_HEIGHT - warpHeight);
```

Comment:
It was never accounted for in the x position because the HUD only takes up vertical space right now.